src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -202,19 +232,21 @@ different reconstruction algorithms (for example ones using motion correction).
 
 Currently supported image contrasts include:
 
-| **Name**  | `contrast_label` | **Description**                                                                                      |
-|---------- | ---------------- | ---------------------------------------------------------------------------------------------------- |
-| BOLD      | bold             | Blood-Oxygen-Level Dependent contrast (specialized T2\* weighting)                                   |
-| CBV       | cbv              | Cerebral Blood Volume contrast (specialized T2\* weighting or difference between T1 weighted images) |
-| Phase     | phase            | Phase information associated with magnitude information stored in BOLD contrast                      |
+| **Name**  | `contrast_label` | **Description**                                                                                                                                                                                                     |
```

Comment:
Should we go ahead with #571 or leave it to another PR?

```suggestion
| **Name**  | `suffix` | **Description**                                                                                                                                                                                                     |
```

Reply:
This will be handled automatically in #610, but I don't see any harm in doing it now as well.